### PR TITLE
doc: 16.2.14 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by zdover23 — APPROVED** [review]

**Review by anthonyeleven — APPROVED** [review]

File: doc/releases/pacific.rst [hunk 1]
```diff
@@ -2,6 +2,209 @@
...
 
+v16.2.14 Pacific
+================
+
+This is the fourteenth backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* CEPHFS: After recovering a Ceph File System post following the disaster recovery
```
Comment (anthonyeleven) [inline doc/releases/pacific.rst:13]: CephFS

**Comment by zdover23** [comment]
>  File "/home/jenkins-build/build/workspace/ceph-pr-docs/build-doc/virtualenv/lib/python3.10/site-packages
> /yaml/constructor.py", line 330, in construct_yaml_timestamp
>    return datetime.date(year, month, day)
> ValueError: month must be in 1..12

**Comment by zdover23** [comment]
Change this line, line 42:

>    released: 2023-30-09

To this line:

>    released: 2023-09-30

Apparently Sphinx and yml are American and not European.

File: doc/releases/releases.yml [hunk 2]
```diff
@@ -38,6 +38,8 @@ releases:
   pacific:
     target_eol: 2023-10-01
     releases:
+      - version: 16.2.14
+        released: 2023-30-09
```
Comment (idryomov) [inline doc/releases/releases.yml:42]: This date can't be right ;)

Comment (yuriw) [reply doc/releases/releases.yml]: fixed typo thx

**Review by nizamial09 — APPROVED** [review]

File: doc/releases/pacific.rst [hunk 3]
```diff
@@ -2,6 +2,209 @@
...
+* src/valgrind.supp: Adding know leaks unrelated to ceph (`pr#49521 <https://github.com/ceph/ceph/pull/49521>`_, Nitzan Mordechai)
+* src/valgrind.supp: Adding know leaks unrelated to ceph (`pr#51341 <https://github.com/ceph/ceph/pull/51341>`_, Nitzan Mordechai)
+* test: correct osd pool default size (`pr#51803 <https://github.com/ceph/ceph/pull/51803>`_, Nitzan Mordechai)
+* test: monitor thrasher wait until quorum (`pr#51799 <https://github.com/ceph/ceph/pull/51799>`_, Nitzan Mordechai)
+* tests: remove pubsub tests from multisite (`pr#48928 <https://github.com/ceph/ceph/pull/48928>`_, Yuval Lifshitz)
+* tools/ceph-dencoder: Fix incorrect type define for trash_watcher (`pr#51778 <https://github.com/ceph/ceph/pull/51778>`_, Chen Yuanrun)
+* tools/ceph-kvstore-tool: fix segfaults when repair the rocksdb (`pr#51254 <https://github.com/ceph/ceph/pull/51254>`_, huangjun)
+* tools/cephfs-data-scan: support for multi-datapool (`pr#50523 <https://github.com/ceph/ceph/pull/50523>`_, Mykola Golub)
+* v16.2.13 (`pr#51404 <https://github.com/ceph/ceph/pull/51404>`_, Ceph Release Team, Patrick Donnelly, Rishabh Dave, Casey Bodley, Radoslaw Zarzynski, Mark Nelson)
```
[the remark this answers is not in the document]
Comment (yuriw) [reply doc/releases/pacific.rst:204]: @ljflores ^

[the remark this answers is not in the document]
Comment (yuriw) [reply doc/releases/pacific.rst]: removed line #204

File: doc/releases/pacific.rst (hunk 1 above)
Comment (vshankar) [reply doc/releases/pacific.rst:13]: We've always used both: CephFS / CEPHFS - its a mix

**Review by vshankar — APPROVED** [review]

File: doc/releases/pacific.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/releases/pacific.rst:13]: Not an acronym :(

Comment (idryomov) [reply doc/releases/pacific.rst:13]: There is a only a single instance of CEPHFS in the release notes (and all of `doc`!), which sneaked in by accident.  Not to bike shed, but it should be CephFS.

File: doc/releases/pacific.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (idryomov) [reply doc/releases/pacific.rst]: Did you mean to also remove "vstart: check mgr status after starting mgr (`pr#51604 <https://github.com/ceph/ceph/pull/51604>`_, Rongqi Sun)" line?

File: doc/releases/pacific.rst (hunk 1 above)
Comment (vshankar) [reply doc/releases/pacific.rst:13]: Hmm.. I do recall seeing more than an instance of it - would have been in a different branch.

**Comment by ljflores** [comment]
Added https://github.com/ceph/ceph/pull/53157 to the list of changes

File: doc/releases/pacific.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (idryomov) [reply doc/releases/pacific.rst]: Should https://github.com/ceph/ceph/pull/51509 be added too -- it was merged last week?

File: doc/releases/pacific.rst (hunk 1 above)
Comment (idryomov) [reply doc/releases/pacific.rst:13]: @yuriw Please address Anthony's nit (CEPHFS -> CephFS).

**Comment by ljflores** [comment]
Hi @idryomov, the PR was merged in accordance with the release date, along with many approvals. The right thing to do here, if these nits are absolutely necessary, is to raise a followup PR. I'll approve it.

**Pull request merged**